lib/semian/net_http.rb

```diff
@@ -61,6 +61,11 @@ def reset_exceptions
       end
     end
 
+    def initialize(*args, semian: true)
+      super(*args)
+      @semian_enabled = semian
```

Comment:
Would it be strange to give this an attribute accessor?

Reply:
> Would it be strange to give this an attribute accessor so that we can toggle it if needed?

From what @casperisfine mentioned, http object might be shared between threads so toggling it isn't thread safe